demos/textfield.html

```diff
@@ -195,6 +205,61 @@
           }, 0);
         </script>
       </section>
+
+      <section class="example">
+        <h2>Textfield - Leading/Trailing icons</h2>
+        <div class="demo-note">
+          <em>
+            Note: Some implementations utilizing leading and trailing icons may wish to set a width on the
+            <code>input</code> element to acheive uniform widths on textfields.
+            This demo acheives that by adding a custom class name: <code>custom-textfield-input</code> and setting
+            some style:
+          </em>
+          <br />
+          <pre>
```

Comment:
Newlines and leading spaces inside the `<pre>` get rendered verbatim, so we might want to collapse it manually so that the output looks correct:

```html
          <pre>.custom-textfield-input {
  width: 300px;
}</pre>
```

Reply:
I see a bunch of extra spaces:

![image](https://user-images.githubusercontent.com/409245/29723871-20ffeb2e-897b-11e7-9498-12e27d149c64.png)

It's not a huge deal - my OCD just doesn't like extraneous whitespace 🙂

Reply:
fixed